src/libsystemd/sd-journal/mmap-cache.c

```diff
@@ -18,34 +18,36 @@
 typedef struct Window Window;
 typedef struct Context Context;
 
+typedef enum WindowFlag {
+        WINDOW_KEEP_ALWAYS  = 1u << (MMAP_CACHE_MAX_CONTEXTS + 0),
+        WINDOW_IN_UNUSED    = 1u << (MMAP_CACHE_MAX_CONTEXTS + 1),
+        WINDOW_INVALIDATED  = 1u << (MMAP_CACHE_MAX_CONTEXTS + 2),
+
+        _WINDOW_UNUSED_MASK = WINDOW_IN_UNUSED - 1,
```

Comment:
this took me a while to grok. 

This is named the wrong way, no? i.e. should be _WINDOWS_USED_MASK, because it contains all bits that indicate that the window is used.

needs a comment. i.e.:

```
/* The mask containing all bits that indicate the windows is currently in use. Covers the all the object types and the additional WINDOW_KEEP_ALWAYS flag */
`` 

or so